src/main/ruby/truffleruby/core/truffle/regexp_operations.rb

```diff
@@ -89,7 +89,7 @@ def self.match_in_region_compare_engines(re, str, from, to, at_start, encoding_c
         md1 = e
       end
       begin
-        md2 = Primitive.regexp_match_in_region(re, str, from, to, at_start, encoding_conversion, start)
```

Comment:
I'll update the method arguments too, so there is no `encoding_conversion` anywhere (since it's always `true` anyway).